crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2084,8 +2084,11 @@ where
             }) => {
                 self.visit_boolean_test(test);
 
-                if flake8_type_checking::helpers::is_type_checking_block(&self.semantic_model, test)
-                {
+                if analyze::typing::is_type_checking_block(&self.semantic_model, test) {
```

Comment:
```suggestion
                if analyze::typing::is_type_checking_block(test, &self.semantic_model {
```

Nit: I would change the order of the arguments so that it reads nicely: The function tests if `test` *is a type checking block*. The semantic model plays a less important role.